packages/kit/test/apps/basics/test/server.test.js

```diff
@@ -22,6 +23,17 @@ test.describe('Content-Type', () => {
 	});
 });
 
+test.describe('CSRF', () => {
+	test('Blocks requests with incorrect origin', async ({ baseURL }) => {
+		const res = await fetch(`${baseURL}/csrf`, {
+			method: 'POST'
+		});
+
+		expect(res.status).toBe(403);
+		expect(await res.text()).toBe('Cross-site POST form submissions are forbidden');
+	});
+});
+
```

Comment:
We're going to have to be cleverer now with this test to make it look more like a form submission.

Reply:
oh ffs, one sec